modules/37_industry/subsectors/equations.gms

```diff
@@ -291,10 +295,8 @@ q37_incinerationEmi(t,regi,sefe(entySe,entyFe),emiMkt)$(
                          entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt)) ..
   vm_incinerationEmi(t,regi,entySe,entyFe,emiMkt)
   =e=
-  (
     v37_plasticWaste(t,regi,entySe,entyFe,emiMkt)
   * pm_incinerationRate(t,regi)
-  ) * (1 - p37_regionalWasteIncinerationCCSshare(t,regi))
```

Comment:
I currently don't understand this - won't this mean that vm_incinerationEmi will always show the FULL carbon content burned, even if part of that is captured? 
if yes, it would be good to adjust the description of vm_incinerationEmi to highlight this.

Reply:
Yes you are absolutely right. And more importantly, we have to update remind2.
We could do this with a postsolve parameter which applies the same capture rate on this as on the rest of the chemicals subsector, but I'd have to also check the implications of the now increased `vm_emiIndCCS`. 
This will take a while, an I'm on holidays next week.

Reply:
Looking forward to `Ctrl+F` through 3000 lines of reportEmi...

Reply:
Its just four lines with `vm_incinerationEmi`, quit the whining.

Reply:
You do it then :)

Reply:
JakobBD/remind#5 & pik-piam/remind2#593